init-letsencrypt.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v docker-compose -f docker-compose.prod.yml)" ]; then
+  echo 'Error: docker-compose -f docker-compose.prod.yml is not installed.' >&2
+  exit 1
+fi
+
+export $(cat .env_prod | grep -v ^\# | xargs); # get variable from .env file
```

Comment:
This should work as well:
```suggestion
. .env_prod # get variable from .env file
```
Where is that file produced?

Reply:
Yeah I changed it back to .env seems like the docker-compose build doesn't load the env_file but load .env default.